content/en/opentelemetry/interoperability/environment_variable_support.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Using OpenTelemetry Environment Variables with Datadog SDKs
+further_reading:
+    - link: '/tracing/trace_collection/library_config/dotnet-core'
+      tag: 'Documentation'
+      text: '.NET Core SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/dotnet-framework'
+      tag: 'Documentation'
+      text: '.NET Framework SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/go'
+      tag: 'Documentation'
+      text: 'Go SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/java'
+      tag: 'Documentation'
+      text: 'Java SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/nodejs'
+      tag: 'Documentation'
+      text: 'Node.js SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/php'
+      tag: 'Documentation'
+      text: 'PHP SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/python'
+      tag: 'Documentation'
+      text: 'Python SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/ruby'
+      tag: 'Documentation'
+      text: 'Ruby SDK Configuration'
+---
+
+After you set up the SDK library with your code and configure the Agent to collect APM data, optionally configure the SDK library as desired using OpenTelemetry environment variables
+
+SDK settings can be configured with the following environment variables with their equivalent Datadog environment variable mapping:
+
+Note: If both set, Datadog environment variables take precedence over OTel environment variables, Likewise Datadog defaults take precedence over OTel defaults. Please refer to the configuration page for the relevant SDK for more information on default values.
+
+## [General SDK Configuration][9]
+
+`OTEL_SERVICE_NAME`
+: **Corresponding Datadog Environment Variable**: `DD_SERVICE`<br>
+Sets the service name<br>
+**Notes**: If `service.name` is also provided in `OTEL_RESOURCE_ATTRIBUTES`, then `OTEL_SERVICE_NAME` takes precedence<br>
+
+`OTEL_LOG_LEVEL`
+: **Corresponding Datadog Environment Variable**: `DD_LOG_LEVEL`<br>
+Log level used by the SDK logger<br>
+**Notes**: A log level of debug will also map to `DD_TRACE_DEBUG=true`<br>
+This maps to `DD_TRACE_LOG_LEVEL` in `node.js`<br>
```

Comment:
Can you double-check that Node is the only that uses `DD_TRACE_LOG_LEVEL` and update this as necessary?

Reply:
PHP also uses DD_TRACE_LOG_LEVEL